src/rgw/rgw_main.cc

```diff
@@ -403,10 +403,7 @@ int main(int argc, const char **argv)
 
       fe = new RGWFCGXFrontend(fcgi_pe, config);
     } else if (framework == "civetweb" || framework == "mongoose") {
-      int port;
-      config->get_val("port", 80, &port);
-
-      RGWProcessEnv env = { store, &rest, olog, port };
+      RGWProcessEnv env = { store, &rest, olog, 0 };
```

Comment:
@mdw-at-linuxbox you can't remove the port here without addressing its use at RGWMongoose::init_env()